app/controllers/general.php

```diff
@@ -202,13 +202,16 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
         ]);
 
         $headers = \array_merge([], $requestHeaders);
+        $executionId = ID::unique();
+        $headers['x-appwrite-execution-id'] = $executionId ?? '';
         $headers['x-appwrite-key'] = API_KEY_DYNAMIC . '_' . $apiKey;
         $headers['x-appwrite-trigger'] = 'http';
         $headers['x-appwrite-user-id'] = '';
         $headers['x-appwrite-user-jwt'] = '';
         $headers['x-appwrite-country-code'] = '';
         $headers['x-appwrite-continent-code'] = '';
         $headers['x-appwrite-continent-eu'] = 'false';
+        $headers['x-appwrite-client-ip'] = $request->getIP();
```

Comment:
@christyjacob4 @eldadfux To confirm header naming is okay

Reply:
We use user almost everywhere, if we start using client, won't we need to change a few more things?

Reply:
We dont have Appwrite User, it could be a guest execution.
I think we need new namespace anyway, and `client` meaning "internet client" makes sense to me.